forge-gui/res/cardsfolder/upcoming/barroom_brawl.txt

```diff
@@ -0,0 +1,7 @@
+Name:Barroom Brawl
+ManaCost:1 G
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | AILogic$ Fight | TgtPrompt$ Choose target creature you control | SubAbility$ DBFight | SpellDescription$ Target creature you control fights target creature you don't control. (Each deals damage equal to its power to the other.)
```

Comment:
SpellDescription$ Target creature you control fights target creature the opponent to your left controls. Then that player may copy this spell and may choose new targets for the copy.

Reply:
Updated SpellDesciption$. Must have been left over from a fight template! Thanks for the catch